Mathlib/Data/List/Defs.lean

```diff
@@ -220,14 +220,13 @@ end mapIdxM
 #align list.sublists List.sublists
 #align list.forall₂ List.Forall₂
 
-/-- `l.All₂ p` is equivalent to `∀ a ∈ l, p a`, but unfolds directly to a conjunction, i.e.
-`List.All₂ p [0, 1, 2] = p 0 ∧ p 1 ∧ p 2`. -/
```

Comment:
As discussed on Zulip, I think not including the ` ∧ True` here is a feature, and probably the main reason this exists in the first place. I'd prefer we leave it be; or at least perform the rename in a separate PR to make it easier to see the impact.

Reply:
Fair enough, will make this a renaming-only PR.